code/controllers/subsystem/processing/obj.dm

```diff
@@ -1,29 +1,5 @@
-SUBSYSTEM_DEF(obj)
+PROCESSING_SUBSYSTEM_DEF(obj)
 	name = "Objects"
 	priority = FIRE_PRIORITY_OBJ
 	flags = SS_NO_INIT
-
-	var/list/processing = list()
-	var/list/currentrun = list()
-
-/datum/controller/subsystem/obj/stat_entry()
-	..("P:[processing.len]")
-
-/datum/controller/subsystem/obj/fire(resumed = 0)
-	if (!resumed)
-		src.currentrun = processing.Copy()
-	//cache for sanic speed (lists are references anyways)
-	var/list/currentrun = src.currentrun
-
-	while(currentrun.len)
-		var/datum/thing = currentrun[currentrun.len]
-		currentrun.len--
-		if(thing)
-			thing.process(wait)
-		else
-			SSobj.processing -= thing
-		if (MC_TICK_CHECK)
-			return
-
-/datum/controller/subsystem/obj/Recover()
-	processing = SSobj.processing
+	wait = 20
```

Comment:
Shouldn't this be 10 to match processing?

Reply:
I'm maintaining its current wait value to avoid changing behavior

Reply:
fair enough, I did confirm that value after I commented, woops